langchain-core/src/messages/tool.ts

```diff
@@ -8,6 +8,16 @@ import {
 } from "./base.js";
 
 export interface ToolMessageFieldsWithToolCallId extends BaseMessageFields {
+  /**
+   * The raw output of the tool.
+   *
+   * **Not part of the payload sent to the model.** Should only be specified if it is
+   * different from the message content, i.e. if only a subset of the full tool output
+   * is being passed as message content.
+   *
+   * `raw_output` is excluded when serializing the tool message.
```

Comment:
Why?

Reply:
was a copy pasta from PY but looks like they dropped that.